src/controllers/list.js

```diff
@@ -722,11 +722,13 @@ class ItemsView {
 
         function play() {
             const currentItem = self.currentItem;
+            const values = self.getSortValues();
 
             if (currentItem && !self.hasFilters) {
                 playbackManager.play({
                     items: [currentItem],
-                    autoplay: true
+                    autoplay: true,
+                    queryOptions: values
```

Comment:
here too

Reply:
here `s/sortValues/values/`

Reply:
Removed.

Thank you for the help :+1:

Reply:
> Removed.

This is `Play All`, right? Here we need to pass sort options so that we play the current folder in the correct order.

Reply:
Yep, you are right. It cannot be removed. I forgot to test that one.

Will be restoring it.

Thank you :-)

Reply:
Restored :-)

Thanks!